app/javascript/components/pxe-customization-template-form/pxe-customization-template-form.schema.js

```diff
@@ -0,0 +1,68 @@
+import { componentTypes } from '@@ddf';
+
+const setFormat = (values) => {
+    return typeof values.type !== 'undefined' && values.type == "CustomizationTemplateKickstart" ? "shell" : "xml";
+};
+
+const createSchema = ( initialValues = {}, loadSchema ) => {
```

Comment:
No need to pass the whole `initialValues` just the `type`, should make your code simpler...